packages/core/src/machine/index.ts

```
@@ -0,0 +1,138 @@
+import { z } from "zod";
+import { Common } from "../common";
+import { createID, fn } from "../utils";
+import { Examples } from "../examples";
+import { machineTable } from "./machine.sql";
+import { getTableColumns, eq, sql, and, isNull } from "../drizzle";
+import { createTransaction, useTransaction } from "../drizzle/transaction";
+
+export module Machine {
```

Biome (1.9.4)

[error] Use the namespace keyword instead of the outdated module keyword.

The module keyword is deprecated to avoid any confusion with the ECMAScript modules which are often called modules.
Safe fix: Use namespace instead.


(lint/suspicious/useNamespaceKeyword)